UnityProject/Assets/Scripts/Objects/Botany/SeedExtractor.cs

```diff
@@ -120,7 +121,7 @@ public void OnDespawnServer(DespawnInfo info)
 		[Server]
 		public void ServerPerformInteraction(HandApply interaction)
 		{
-			var grownFood = interaction.HandObject?.GetComponent<GrownFood>();
+			var grownFood = interaction.HandObject.GetComponent<GrownFood>();
```

Comment:
``interaction.HandObject`` can be null if the object is interacted with an empty hand. You could avoid checking for it by making ``interaction.HandObject != null`` a condition for this interaction to trigger. You would need to make this ``ICheckedInteractable`` instead of just ``IInteractable``

Reply:
if you have any question on how to do that, just ask. We're happy to help.

Reply:
I think I've got it. I just have to test it out, I'll let you know how I go.